tests/integration_tests/test_hyperopt.py

```diff
@@ -314,3 +315,96 @@ def test_hyperopt_run_hyperopt(csv_filename, search_space, tmpdir, ray_cluster):
 
     if os.path.isfile(os.path.join(tmpdir, "test_hyperopt", "hyperopt_statistics.json")):
         os.remove(os.path.join(tmpdir, "test_hyperopt", "hyperopt_statistics.json"))
+
+
+@pytest.mark.distributed
+@pytest.mark.parametrize("search_space", ["random"])
+def test_hyperopt_run_hyperopt_with_shared_params(csv_filename, search_space, tmpdir, ray_cluster):
```

Comment:
It looks like you aren't using the `ray_cluster` argument?

Also, instead of passing in `tmpdir`, can you use the [tempfile](tmpdir) package? You should be able to remove the cleanup at the end of the tests.